<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostelersForm</class>
 <widget class="QDialog" name="HostelersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>451</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="roomL">
      <property name="text">
       <string>Room:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="room"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="roomE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="rollL">
      <property name="text">
       <string>Roll No:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="roll"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="rollE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="nameL">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="name"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="nameE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="branchL">
      <property name="text">
       <string>Branch:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="branch"/>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="branchE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="phoneL">
      <property name="text">
       <string>Phone:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="phone"/>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="phoneE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="emailL">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="email"/>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="emailE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="emergencyL">
      <property name="text">
       <string>Emergency Contact:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="emergency"/>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="emergencyE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="remark"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="remarkL">
      <property name="text">
       <string>Remark:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="remarkE">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
